packages/accounts-controller/tsconfig.build.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.packages.build.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist",
+    "rootDir": "./src"
+  },
+  "references": [
+    {
+      "path": "../base-controller/tsconfig.build.json"
+    },
+    {
+      "path": "../controller-utils/tsconfig.build.json"
+    },
```

Comment:
Do we still need `@metamask/controller-utils`? I don't think it's listed in the dependencies

Reply:
You're right, this can be removed now. Changed in 27ae5f33b1cf2d44a78c9c0100bc1f529d14086a